server/utils.py

```diff
@@ -6,41 +6,38 @@
 
 from glob import glob
 
-from flask import jsonify, make_response, g
+from flask import jsonify, make_response
 
-from server import app
+from server import app, db
+from models import User, Access, File
 
 
 CHUNK_PREFIX = 'chunk.'
 
 
 def generate_auth_token(server_token, user, name=None, email=None, duration_days=1):
     auth_token = base64.urlsafe_b64encode(os.urandom(12))
+    server_name = app.config['SERVER_TOKENS'][server_token]['name']
+    server_id = app.config['SERVER_TOKENS'][server_token]['id']
```

Comment:
This will crash with a KeyError if the server_token is invalid, instead of a proper invalid authentication response.